lib/internal/crypto/cipher.js

```diff
@@ -154,8 +154,20 @@ ObjectSetPrototypeOf(Cipher.prototype, LazyTransform.prototype);
 ObjectSetPrototypeOf(Cipher, LazyTransform);
 
 Cipher.prototype._transform = function _transform(chunk, encoding, callback) {
-  this.push(this[kHandle].update(chunk, encoding));
-  callback();
+  let error = null;
+  try {
+    const inputEncoding = encoding || getDefaultEncoding();
+    if (typeof chunk === 'string') {
+      validateEncoding(chunk, inputEncoding);
```

Comment:
nit: we can avoid calling `getDefaultEncoding` when chunk is not a string
```suggestion
    if (typeof chunk === 'string') {
      encoding ||= getDefaultEncoding();
      validateEncoding(chunk, encoding);
```

Reply:
This also happens on Node 14, and if IIRC ||= doesn't work on Node 14 - will we want to backport this?

Reply:
I guess a `if` statement would work just as fine.
```suggestion
    if (typeof chunk === 'string') {
      if(!encoding) encoding = getDefaultEncoding();
      validateEncoding(chunk, encoding);
```